src/registry/models.py

```diff
@@ -1,7 +1,26 @@
+import os
 from django.conf import settings
 from django.db import models
 
 
+def get_plangs_path() -> str:
+    return os.path.join(settings.STATIC_ROOT, "img/plangs")
+
+
+class ImplementationLanguage(models.Model):
+    language = models.CharField(max_length=100, null=False, blank=False, unique=True)
+    svg_path = models.FilePathField(path=get_plangs_path(), match=".svg$", null=True)
```

Comment:
FilePathField is an interesting field, it will ensure that every `svg_path` value is in the `path` provided and `match` will filter only the svg ones. (obs: the repeated ones are automatically generated by collectstatic)

![Uploading image.png…]()